deployment/helm/iot-edge-l2/templates/system-modules/opc-publisher-module.yaml

```diff
@@ -28,6 +28,7 @@ spec:
         dapr.io/enabled: "true"
         dapr.io/app-id: "opc-publisher-module"
         dapr.io/app-protocol: "grpc"
+        dapr.io/log-level: "debug"
```

Comment:
Is this a leftover from local debugging?

Reply:
Indeed, thanks for catching.